nunit.sln

```diff
@@ -1,6 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 2012
+# SharpDevelop 5.1
```

Comment:
This is just Sharpdevelop I believe. I like it as it is lightweight compared to [VS Code](https://en.wikipedia.org/wiki/Visual_Studio_Code) is looking interesting.